app/app.go

```diff
@@ -135,12 +185,15 @@ var (
 		mint.AppModuleBasic{},
 		distr.AppModuleBasic{},
 		gov.NewAppModuleBasic(
-			paramsclient.ProposalHandler,
-			distrclient.ProposalHandler,
-			upgradeclient.ProposalHandler,
-			upgradeclient.CancelProposalHandler,
-			ibcclientclient.UpdateClientProposalHandler,
-			ibcclientclient.UpgradeProposalHandler,
+			append(
```

Comment:
Why do we use `append` here? Can't we simply list the handlers as they were before?

Reply:
@bragaz Can we go back to not using `append`?

Reply:
@bragaz Got it, thanks